selfdrive/controls/controlsd.py

```diff
@@ -580,7 +576,7 @@ def publish_logs(self, CS, start_time, actuators, v_acc, a_acc, lac_log):
     controlsState.enabled = self.enabled
     controlsState.active = self.active
     controlsState.curvature = curvature
-    controlsState.steeringAngleDesiredDeg = angle_steers_des
+    controlsState.steeringAngleDesiredDeg = actuators.steeringAngleDeg
```

Comment:
deprecate the field?

Reply:
We don't want it?

Reply:
duplicating fields across packets generally isn't great. any reason to keep it?

Reply:
ah I see its already in carcontrol, ok!